src/Umbraco.Web.BackOffice/Controllers/ContentTypeController.cs

```diff
@@ -248,13 +248,13 @@ public IActionResult GetWhereCompositionIsUsedInContentTypes(GetAvailableComposi
         }
 
         [Authorize(Policy = AuthorizationPolicies.TreeAccessAnyContentOrTypes)]
-        public ContentPropertyDisplay GetPropertyTypeScaffold(int id)
+        public ActionResult<ContentPropertyDisplay> GetPropertyTypeScaffold(int id)
         {
             var dataTypeDiff = _dataTypeService.GetDataType(id);
 
             if (dataTypeDiff == null)
             {
-                throw new HttpResponseException(HttpStatusCode.NotFound);
+                return new ValidationErrorResult(dataTypeDiff, StatusCodes.Status404NotFound);
```

Comment:
This shoud not be a ValidationErrorResult
```suggestion
                return NotFound();
```